Spanner/src/Transaction.php

```diff
@@ -96,7 +101,8 @@ public function __construct(
         Session $session,
         $transactionId = null,
         $isRetry = false,
-        $tag = null
+        $tag = null,
+        $options = null
```

Comment:
1. Options should default to `[]` (See other places this is implemented, e.g. `Snapshot`)
2. Options should have consistent and well-documented parameters, e.g. `id` for "transaction ID". This is from `Snapshot`:
```php
    /**
     * @param array $options [optional] {
     *     Configuration Options.
     *
     *     @type string $id The Transaction ID. If no ID is provided,
     *           the Transaction will be a Single-Use Transaction.
     *     @type Timestamp $readTimestamp The read timestamp.
     * }
     */
```

Reply:
Sure, will take care of the docblock once we decide on the design.

Reply:
It's difficult to review the current design when the options parameters are not documented. As it does not take very long to document the parameters that you intend to support in your design, it's important to include this now, even while still deciding on the design.